sections/dashboard/EligibilityBox/EligibilityBox.tsx

```diff
@@ -0,0 +1,90 @@
+import { useCallback, useState } from 'react';
+import styled from 'styled-components';
+
+import Button from 'components/Button';
+import Input from 'components/Input/Input';
+import Text from 'components/Text';
+
+import ELIGIBILITY_SET from './EligibilitySet';
+import LONOCLAUSE_SET from './LonoClause';
+
+const EligibilityBox = () => {
+	const [address, setAddress] = useState('');
+	const [isEligible, setIsEligible] = useState<boolean>();
+	const [isLonoEligible, setIsLonoEligible] = useState<boolean>();
+
+	const handleChangeAddress = useCallback((e) => {
+		setAddress(e.target.value);
+	}, []);
+
+	const handleCheck = useCallback(() => {
+		setIsEligible(!!(address && ELIGIBILITY_SET.has(address)));
+		setIsLonoEligible(!!(address && LONOCLAUSE_SET.has(address)));
+	}, [address]);
+
+	return (
+		<div style={{ maxWidth: '445px' }}>
+			<EligibilitySectionTitle>SNX Staker & Synth Trader Distribution</EligibilitySectionTitle>
+			<EligibilityContainer>
+				<EligibilityText>Enter your wallet address to check eligibility</EligibilityText>
+				<EligibilityInput placeholder="0x..." value={address} onChange={handleChangeAddress} />
+				<Button fullWidth size="sm" onClick={handleCheck}>
+					Check
+				</Button>
+			</EligibilityContainer>
+			{isEligible !== undefined && !isLonoEligible && (
+				<EligibilityStatus $isEligible={isEligible}>
+					{`Address is ${!isEligible ? 'not' : ''} eligible for the Aelin pool (Nov 1 - Nov 14)`}
```

Comment:
Can we add one more condition here to show "Visit pool" at the end of the sentence (add a dot to the end of the first sentence, too) and link this out to the pool on Aelin (once live/set up) _if_ address is eligible

Reply:
Good idea.